src/openrct2/scenes/Scene.cpp

```diff
@@ -31,19 +31,13 @@ GameState_t& Scene::GetGameState()
 
 void Scene::FinishScene()
 {
-    if (_nextScene != nullptr)
+    if (_onFinish != nullptr)
     {
-        _context.SetActiveScene(_nextScene);
-        _nextScene = nullptr;
+        _onFinish();
     }
 }
 
-IScene* Scene::GetCompletionScene()
+void Scene::SetOnComplete(std::function<void()> onFinish)
```

Comment:
This is a bit inconsistent. Probably better to rename `onFinish` to `onComplete` as well.

Reply:
The function name should start with a lower case letter as well (in line with the rules from OpenLoco that we recently adopted).

Reply:
Ah, you're right. We probably should've taken that into account when I ported the scenes in the first place, as the new code style had already taken effect by then. 

For now, I'd prefer the functions to at least be capitalised consistently, rather than having one odd one out. We can (and should) adjust all of them at once, but that is out of scope for this PR, imo.